sqlmesh/core/model/decorator.py

```diff
@@ -29,15 +29,19 @@ def __init__(self, name: str, is_sql: bool = False, **kwargs: t.Any) -> None:
         self.is_sql = is_sql
         self.kwargs = kwargs
 
-        # Make sure that argument values are expressions in order to
-        # pass validation in ModelMeta.
+        # Make sure that argument values are expressions in order to pass validation in ModelMeta.
         calls = self.kwargs.pop("audits", [])
         self.kwargs["audits"] = [
             (call, {})
             if isinstance(call, str)
             else (
                 call[0],
-                {arg_key: exp.convert(arg_value) for arg_key, arg_value in call[1].items()},
+                {
+                    arg_key: exp.convert(
+                        tuple(arg_value) if isinstance(arg_value, list) else arg_value
```

Comment:
Very nice! I hope this was the only reason 🤞 Good place to fix this otherwise.

Reply:
I hope so too! 🤞 😅